app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CardsMailer < ApplicationMailer
+  default from: Rails.application.secrets.smtp_email_address
+
+  def pending_cards_notification(user)
+    @name = user.name
+
+    mail(to: user.email, subject: 'У вас появились карточки для повторения')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.